hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -409,6 +409,60 @@ def hpat_pandas_series_groupby_impl(
     return hpat_pandas_series_groupby_impl
 
 
+@overload_method(SeriesType, 'isna')
+def hpat_pandas_series_isna(self):
+    """
+    Pandas Series method :meth:`pandas.Series.append` implementation.
+
+    .. only:: developer
+
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_isna
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_isna1
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_str_isna1
+
+    Parameters
+    -----------
+    self : :obj:`pandas.Series` object
+               input argument
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
+    """
+
+    _func_name = 'Method isna().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError(
+            '{} The object must be a pandas.series. Given self: {}'.format(_func_name, self))
+
+    if isinstance(self.dtype, (types.Integer, types.Float)):
+
+        def hpat_pandas_series_isna_impl(self):
+
+            return pandas.Series(numpy.isnan(self._data))
+
+        return hpat_pandas_series_isna_impl
+
+    if isinstance(self.dtype, types.UnicodeType):
+
+        def hpat_pandas_series_isna_impl(self):
```

Comment:
What is the algo here? Is it copied from somewhere? Please put a comment about it